extra/resources/ClusterMon.in

```diff
@@ -43,15 +43,15 @@ It outputs current cluster status to the html.
 
 <parameters>
 
-<parameter name="user">
+<parameter name="user" reloadable="1">
```

Comment:
I just realized anything used in CMON_CMD is not reloadable, because ClusterMon_start() uses it to run crm_mon as a daemon. To change the user it's running as (or anything else in CMON_CMD), we have to stop and start crm_mon again.

Before, I was thinking the recurring monitor updated the HTML file, but it's actually the crm_mon started by the start action. The recurring monitor only checks that crm_mon is still running.

Which I guess means that there's nothing reloadable ...